editor/store/actions.js

```diff
@@ -252,6 +253,20 @@ export function autosave() {
 	};
 }
 
+export function isAutosaving( isAutosaving ) {
```

Comment:
I think the actions are better named as "verbs", maybe `toggleAutosave`?

Reply:
thanks, good suggestion

Reply:
changed